lua/xadmin/_config/sh_permissions.lua

```diff
@@ -0,0 +1,64 @@
+/*=============
+Permission Config
+=============*/
+-- This section of the config allows you to change what power levels
+-- can access different permissions from the base xAdmin addon
+
+-- to change what level can access Admin Chat see the option
+-- 'xAdmin.Config.AdminChat' above!
+-- Note: This also effects all aliases of the command
+xAdmin.Config.PowerlevelPermissions = {
```

Comment:
I would add a default power when one isn't provided (If it's not included here but is used in the command, it will pass nil). An easy way to do this is to check in `xAdmin.Core.RegisterCommand` [here](https://github.com/TheXYZNetwork/xAdmin/blob/master/lua/xadmin/core/sv_commands.lua#L1). You can just validate power and then make it some kind of default if it isn't provided. (Maybe a default set here? Or the power that can see staff chat?)